kalite/distributed/features/steps/exercises.py

```diff
@@ -9,7 +8,7 @@
 
 @given("I open an exercise")
 def step_impl(context):
-    context.browser.get(context.browser_url("/"))
+    pass
```

Comment:
No plan to implement this at this point. More of an example than anything. The backbone stuff is hard to interact with using Selenium presumably because it wasn't designed with that in mind.